x-pack/test/api_integration/apis/ml/data_frame_analytics/get_spaces.ts

```diff
@@ -140,5 +153,45 @@ export default ({ getService }: FtrProviderContext) => {
     it('should fail to list jobs stats by job ids if one of them is in a different space', async () => {
       await runRequest(idSpace1, 404, true, `${jobIdSpace1},${jobIdSpace2}`);
     });
+
+    describe('GetDataFrameAnalyticsIdMap with spaces', () => {
+      it(`should return a map of objects from ${idSpace1} leading up to analytics job id created in ${idSpace1}`, async () => {
+        const body = await runMapRequest(idSpace1, 200, jobIdSpace1);
+
+        expect(body).to.have.keys('elements', 'details', 'error');
```

Comment:
Would be good to also check that the `error` value is `null`.

Reply:
Updated in 08bde443ff73808696c65900e93c6bf22fa70265